src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1465,6 +1506,8 @@ def stop_servers(self):
                 errors.append(
                     "ERROR: At least one multi-variant server was not found in "
                     "its expected state; stopping all servers")
+                # dump engines stacks if not already done
+                self.dump_engines_stacks("Some engine not in expected state")
```

Comment:
Do not see when this method call is needed. From what I understand `error()`, `fail()` and `report_timeout()` methods overloading should be sufficient.

Reply:
There are cases where a test status can be successful but some server/engine is found in an unexpected state during tearDown()

Reply:
OK, thanks for the info.